docker-compose.override.yml

```diff
@@ -2,7 +2,7 @@ version: "3"
 
 services:
   organization-catalogue:
-    image: brreg/organization-catalogue:latest
+    image: eu.gcr.io/fdk-infra/organization-catalogue:env_ut1_latest
```

Comment:
Er det i det hele tatt noe vits med override fil? Kan dette flyttes inn i hoved docker-compose?

Reply:
Override brukes kun av fdk-docker-compose. Jeg er for at dette flyttes dit, så kan jeg slette override-fila. Men frem til det lar jeg det være som dette